RecoLocalMuon/CSCSegment/src/CSCSegAlgoRU.cc

```diff
@@ -128,13 +133,23 @@ std::vector<CSCSegment> CSCSegAlgoRU::buildSegments(const CSCChamber* aChamber,
     if(aState.doCollisions && search_disp && int(rechits.size()-used_rh)>2){//check if there are enough recHits left to build a segment from displaced vertices
       aState.doCollisions = false;
       aState.windowScale = 1.; // scale factor for cuts
-      aState.dRMax = 4.0;
-      aState.dPhiMax = 4*aState.dPhiMax;
-      aState.dRIntMax = 4*aState.dRIntMax;
-      aState.dPhiIntMax = 4*aState.dPhiIntMax;
-      aState.chi2Norm_2D_ = 10*aState.chi2Norm_2D_;
-      aState.chi2_str_ = 200;
-      aState.chi2Max = 4*aState.chi2Max;
+      if(enlarge){ 
+	aState.dRMax = 4.0;
+	aState.dPhiMax = 4*aState.dPhiMax;
+        aState.dRIntMax = 4*aState.dRIntMax;
+        aState.dPhiIntMax = 4*aState.dPhiIntMax;
+        aState.chi2Norm_2D_ = 10*aState.chi2Norm_2D_;
+        aState.chi2_str_ = 200;
+        aState.chi2Max = 4*aState.chi2Max;
+      } else {
+        aState.dRMax = 2.0;
+	aState.dPhiMax = 2*aState.dPhiMax;
```

Comment:
please do not copy-paste essentially the same lines of code.
Define a scale constant (1. for enlarge=false; and 2. for enlarge = true)